typedoc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
`treatWarningsAsErrors` is removed due to a new version warning about unresolved types

![image](https://github.com/waku-org/js-waku/assets/118575614/13c7e4f6-82e1-4b7b-bf13-2a80aaf4ee3a)

another important change is some comments had to be changed because links were broken in the generated docs

Reply:
I am also confused here. Seems like you are fixing the warnings (broken links) so shouldn't we keen `treatWarningAsErrors`?

Reply:
> i don't think we should remove a flag because its leading to warnings/errors
it's leading only to warnings, not errors

the disadvantage of doing so it that it can happen that link in the generated docs won't be resolved
for particular cases here - all links **are** getting resolved though there is a warning for them 

I created a questing to clarify this problem here - https://github.com/TypeStrong/typedoc/issues/2403

For now I propose to move forward and address in a following PR once it is clarified

@waku-org/js-waku-developers

Reply:
I disagree that it is a priority but definitely worth fixing in the future. 
https://github.com/waku-org/js-waku/issues/1593